catroid/src/androidTest/java/org/catrobat/catroid/test/physics/actions/IfOnEdgeBouncePhysicsActionTest.java

```diff
@@ -142,8 +141,6 @@ public void testSpriteOverlapsRightAndTopAxis() {
 		float setVelocityYValue = 400.0f;
 		physicsObject.setVelocity(setVelocityXValue, setVelocityYValue);
 
-		assertEquals(setXValue, sprite.look.getX());
-		assertEquals(setYValue, sprite.look.getY());
```

Comment:
Nice!
But instead of deleting the assertions with wrong parameters, use the correct expected value, please.